.github/workflows/ci.yml

```diff
@@ -86,8 +93,13 @@ jobs:
         env:
           TEST_KAFKA_HOST: kafka
           TEST_KAFKA_PORT: 29092
-          TEST_MYSQL_HOST: sql
-          TEST_MYSQL_PORT: ${{ job.services.mysql.ports['3306'] }}
+          TEST_MYSQL_HOST: mysql
+          TEST_MYSQL_PORT: 3306
+          TEST_POSTGRES_PASSWORD: postgres
+          TEST_POSTGRES_USER: postgres
+          TEST_POSTGRES_HOST: postgres
+          TEST_POSTGRES_PORT: 5432
+          TEST_POSTGRES_DB: postgres
```

Comment:
I had a heck of a time getting the tests to run until I set these ports to their default ports. I was unable to convince the mysql tests to find the mysql container when I added the _postgres_ container - suggesting to me that something weird was going on with docker networking and Actions. But this fix worked, and I figured it's ~fine.